quicktests/overlaying/tests/realistic/basic_pie_real.js

```diff
@@ -52,10 +52,23 @@ function run(svg, data, Plottable) {
   ABplot.addDataset(new Plottable.Dataset(data[2]));
   ABplot.sectorValue(function(d) { return d.percent; });
   ABplot.attr("fill", function(d) { return d.region; }, colorScale);
-  ABplot.innerRadius(50);
-  ABplot.outerRadius(100);
+  ABplot.attr("stroke", "#000000");
+  ABplot.innerRadius(45);
+  ABplot.outerRadius(90);
   var ABGroup = new Plottable.Components.Group([ABplot, ABlabel]);
 
+  new Plottable.Interactions.Click()
+  .onClick(function(p){
+    ABplot.entities().forEach(function(e){
+      e.selection.attr("stroke-width", 0);
```

Comment:
If no slice is selected this removes the stroke from every slice, so that behavior was a bit unexpected (I clicked in the center of the hole where the label was).